### PR TITLE
Feat: Create workflow to RequestOutFitWindowPacket.

### PR DESCRIPTION
**Pull request by MarcusSilvaDev**
Previews:
image
image
image
image
image

### CODE REVIEW

File: src/Data/NeoServer.Data/Seeds/PlayerModelSeed.cs [hunk 1]
```diff
@@ -167,7 +167,7 @@ public static void Seed(EntityTypeBuilder<PlayerModel> builder)
                 Health = 4440,
                 MaxHealth = 4440,
                 Vocation = 3,
-                Gender = Gender.Male,
```
Comment (caioavidal) [inline src/Data/NeoServer.Data/Seeds/PlayerModelSeed.cs:170]: can you ignore this file changes?

File: src/Game/NeoServer.Game.Common/Creatures/Players/Gender.cs [hunk 2]
```diff
@@ -2,6 +2,6 @@
 
 public enum Gender : byte
 {
-    Male,
-    Female
+    Female,
+    Male
```
Comment (caioavidal) [inline src/Game/NeoServer.Game.Common/Creatures/Players/Gender.cs]: Can we have a byte assigned to the type?
ie:  Male = 0, Female = 1

**Review by caioavidal — CHANGES_REQUESTED** [review]

File: src/Game/NeoServer.Game.Common/Creatures/Players/Gender.cs (hunk 2 above)
Comment (MarcusSilvaDev) [reply src/Game/NeoServer.Game.Common/Creatures/Players/Gender.cs]: It is possible, but currently to the outfit Object, the values are inverted.  We need to find a solution if there are behaviors different from this.

**ie: Female = 0, Male = 1**

Comment (caioavidal) [reply src/Game/NeoServer.Game.Common/Creatures/Players/Gender.cs]: I think we can change it to: Female = 0, Male = 1 then.
this is probably only used by outfits

**Review by caioavidal — APPROVED** [review]

**Pull request merged**